crates/bevy_ui/src/focus.rs

```diff
@@ -52,96 +48,95 @@ pub fn ui_focus_system(
         Option<&FocusPolicy>,
     )>,
 ) {
-    if let Some(cursor_moved) = state.cursor_moved_event_reader.latest(&cursor_moved_events) {
-        state.cursor_position = Some(cursor_moved.position);
-    }
-    if let Some(touch) = touches_input.get_pressed(0) {
-        state.cursor_position = Some(touch.position());
-    }
+    let cursor_position = if let Some(cursor_position) = windows
+        .get_primary()
+        .and_then(|window| window.cursor_position())
```

Comment:
will that work when having multiple windows? anyway did it even work before with multiple windows?

Reply:
It didn't filter according to WindowId, so it would have supported multiple windows for focus events. But the rest of the UI system doesn't support multiple windows, so it doesn't really matter. If/when that ever gets built we can use the same approach here.